.nvmrc

```diff
@@ -0,0 +1 @@
+lts/*
```

Comment:
No, I checked - `nvm` only looks in the one place for it (at the project level).

Reply:
Why is the file needed? Can we pass command line args instead?

Reply:
It's used by nvm to indicate what version of node we should be using on a project-- so as a dev, you can jump into a project and just type `nvm use` and you know you're getting the correct node version. We could specify something for travis, but then it looses the developer helpfulness part :)

Reply:
Since we can easily avoid it for Travis, and it often works fine w/ any recent version, and it's not hard to test a few when it doesn't, I'd vote for not cluttering the root w/ another file.

If you and @coreymckrill are in favor, though, I can disagree and commit.

Reply:
FWIW, it appears that in the [original issue](https://github.com/nvm-sh/nvm/issues/110) that spawned the .nvmrc file, a lot of people didn't want the extra file either, and would have preferred just using a value in package.json. Obviously that's not what ended up happening, and I couldn't quite figure out how things played out reading through the issue comments.

Given that variations in a developer's node version can change the package-lock file and/or cause some of our package scripts to not run correctly, it seems worthwhile to specify the correct node version for our project, and the .nvmrc file is a common convention that contributors are likely to understand, so I'm in favor.